File: Backend/MoneyService/WebApplication/Models/UserMe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using WebApplication.Models;
using WorkWithDatabase.Models;

namespace WebApplication.DTO
{
    public class UserMe
    {
        public UserMe(string username, string photo, UserStatus status)
        {
            Username = username;
            Photo = photo;
            Status = status;
        }
        public string Username { get; set; }
        public string Photo { get; set; }
        public UserStatus Status { get; set; }
    }
}

File: Android/ModulBank/app/src/main/java/com/alex/modulbank/DTO/UserMe.kt
package com.alex.modulbank.DTO

import com.google.gson.annotations.SerializedName

data class UserMe (
    @SerializedName("username") val username: String,
    @SerializedName("status")   val status: UserStatus,
    @SerializedName("photo")    val photo: String,
    @SerializedName("email")    var email: String=""
)